core/input/input_map.cpp

```diff
@@ -85,6 +85,38 @@ String InputMap::suggest_actions(const StringName &p_action) const {
 	return error_message;
 }
 
+#ifdef TOOLS_ENABLED
+void InputMap::get_argument_options(const StringName &p_function, int p_idx, List<String> *r_options) const {
+	const String pf = p_function;
+	bool first_argument_is_action = false;
+	if (p_idx == 0) {
+		first_argument_is_action = (pf == "has_action" || pf == "erase_action" ||
+				pf == "action_set_deadzone" || pf == "action_get_deadzone" ||
+				pf == "action_has_event" || pf == "action_add_event" || pf == "action_get_events" ||
+				pf == "action_erase_event" || pf == "action_erase_events");
+	}
+	if (first_argument_is_action || (p_idx == 1 && pf == "event_is_action")) {
+		// Cannot be done. Gets the options in the context of the Editor (no user-defined actions).
+		//for (const StringName &E : get_actions()) {
+		//	r_options->push_back(E.operator String().quote());
+		//}
```

Comment:
```suggestion
		// Cannot rely on `get_actions()`, as it gets the options in the context of the Editor (no user-defined actions).
```
Avoid dead code that we'll keep wanting to remove ;)